phpcs.xml

```diff
@@ -20,4 +20,14 @@
     <rule ref="LaminasCodingStandard"/>
     <exclude-pattern>test/TestAsset/**</exclude-pattern>
     <exclude-pattern>test/**/TestAsset/**</exclude-pattern>
+
+    <!-- Needed to avoid breaking code for 7.4 since $instance::class only works in PHP 8.0 -->
+    <rule ref="SlevomatCodingStandard.Classes.ModernClassNameReference.ClassNameReferencedViaFunctionCall">
```

Comment:
For what are these necessary? Not sure if I understand this change, I think it has to do with the fact that you were running code-style checks with PHP > 8.0, so please ensure that you execute codestyle locally with the lowest supported PHP version of this component (which is actually PHP 7.4 as per `composer.json`.

Reply:
> I think it has to do with the fact that you were running code-style checks with PHP > 8.0

Ah that is indeed the case. completely missed to rebuild the images I use for development with 7.4 before performing these fixes. will review the second commit and try to minimize the changes. Thanks for pointing this out.